.github/workflows/devbuild.yml

```diff
@@ -113,11 +113,22 @@ jobs:
         #
         # In short, OpenGL libraries and headers are required. Without
         # installing this package, cmake won't find out the correct lib path.
-        if [ "${{ matrix.os }}" == "ubuntu-22.04" ] ; then \
-          sudo apt-get -qy install \
-              qt6-3d-dev xvfb \
-              libgl1-mesa-dev
-        fi
+        # if [ "${{ matrix.os }}" == "ubuntu-22.04" ] ; then \
```

Comment:
Please add comments to explain why the block of commands is commented out.

Reply:
I don't like the style of comments:

```
# # ...
```

It unnecessarily makes the lines longer.

And please write "... cmake won't find the correct ...".  It seems "find out" isn't what you meant.